ui/StatusQ/include/StatusQ/aggregator.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <QAbstractListModel>
```

Comment:
Should be removed to limit number of imports in headers, forward declaration can be used instead (include should go to cpp and for `QAbstractItemModel`, not `QAbstractListModel`)

```
class QAbstractItemModel;
```
below includes